### PR TITLE
Only get anime rss feed if user has anime shows

### PR DESCRIPTION
**Pull request by fernandog**
TODO: pop anime from here:
https://github.com/SickRage/SickRage/blob/develop/sickbeard/providers/kat.py#L64

and any other provider that get anime rss feed

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/helpers.py [hunk 1]
```diff
@@ -1793,3 +1793,14 @@ def is_ip_private(ip):
     priv_20 = re.compile(r"^192\.168\.\d{1,3}.\d{1,3}$")
     priv_16 = re.compile(r"^172.(1[6-9]|2[0-9]|3[0-1]).[0-9]{1,3}.[0-9]{1,3}$")
     return priv_lo.match(ip) or priv_24.match(ip) or priv_20.match(ip) or priv_16.match(ip)
+
+def anyAnimeShow()
+    try:
+        main_db_con = db.DBConnection()
+        sql_results = main_db_con.select("SELECT distinct anime FROM tv_shows order by anime desc limit 1")
```
Comment (miigotu) [inline sickbeard/helpers.py]:
``` sql
SELECT anime FROM tv_shows WHERE anime != 0 LIMIT 1
```

File: sickbeard/helpers.py [hunk 2]
```diff
@@ -1793,3 +1793,14 @@ def is_ip_private(ip):
...
+        if sql_results:
```
Comment (miigotu) [inline sickbeard/helpers.py]: return len(sql_results) > 0

File: sickbeard/helpers.py (hunk 1 above)
Comment (fernandog) [inline sickbeard/helpers.py]: @miigotu 
boolean(null) = false?

Comment (miigotu) [inline sickbeard/helpers.py]: anime column is always 0,1,2,3

File: sickbeard/helpers.py (hunk 2 above)
Comment (fernandog) [inline sickbeard/helpers.py]: need to go. can you do it? understand what I'm trying to do? agree?

Comment (miigotu) [inline sickbeard/helpers.py]: i understand what you are trying to do, but I'm 50/50 on whether it actually needs done. I have some other stuff I am behind on that I need to take care of before I can do this if you want me to do it.

File: sickbeard/helpers.py [hunk 3]
```diff
@@ -1793,3 +1793,14 @@ def is_ip_private(ip):
     priv_20 = re.compile(r"^192\.168\.\d{1,3}.\d{1,3}$")
     priv_16 = re.compile(r"^172.(1[6-9]|2[0-9]|3[0-1]).[0-9]{1,3}.[0-9]{1,3}$")
     return priv_lo.match(ip) or priv_24.match(ip) or priv_20.match(ip) or priv_16.match(ip)
+
+def anyAnimeShow():
```
Comment (medariox) [inline sickbeard/helpers.py:1797]: Please use Python naming conventions: https://www.python.org/dev/peps/pep-0008/#function-names

Comment (miigotu) [inline sickbeard/helpers.py:1797]: and tbh, that method should probably be in the generic provider instead, so other providers can use it.

Comment (fernandog) [inline sickbeard/helpers.py:1797]: Possible to run only once each dailysaerch thread or backlosearch or manual serch?

Comment (miigotu) [inline sickbeard/helpers.py:1797]: Thats why I'm 50/50 on whether it should actually be done. Adding a bunch of code and continually trying to work around it just to save one request to a public provider doesnt really change anything. It saves a minute amount of processing and some log lines. And the SQL query would negate the savings of that processing. It doesnt hurt to just leave it as it is.

**Comment by fernandog** [comment]
Close if not worth then

**Comment by fernandog** [comment]
@miigotu should I close this or worth to try make it work ?

**Comment by miigotu** [comment]
Too much hassle for too little gain

**Pull request closed without merging**